Makefile

```diff
@@ -87,13 +92,14 @@ selfcheck: ## check that the Makefile is well-formed
 
 ## Localization targets
 
-extract_translations: ## extract strings to be translated, outputting .mo files
-	rm -rf docs/_build
-	cd platform_plugin_ontask && ../manage.py makemessages -l en -v1 -d django
-	cd platform_plugin_ontask && ../manage.py makemessages -l en -v1 -d djangojs
+extract_translations: ## extract strings to be translated, outputting .po files
+	cd $(PACKAGE) && i18n_tool extract
+	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
+	if [ -f "$(EXTRACTED_DJANGOJS)" ]; then cat $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT); rm $(EXTRACTED_DJANGOJS); fi
```

Comment:
I'd prefer if we keep this as is, and then figure out the best way of adding translations in a plugin like this. But let's follow the standard we've used so far. What do you think?

Reply:
We could leave it as it is now in the [cookiecutter]( https://github.com/openedx/edx-cookiecutters/blob/a6a80787b102d9811aef4432bd3b709603cf720f/cookiecutter-django-app/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile#L86-L91), what do you think?

Reply:
Agreed!